lmnet/lmnet/utils/box.py

```diff
@@ -0,0 +1,125 @@
+import numpy as np
+
+
+def iou(boxes, box):
+    """Calculate overlap
+
+    Args:
+        boxes: boxes in the image. shape is [num_boxes, 4 or more(x, y, w, h, ...)]
+        box: a single box in the image. shape is [4 or more (x, y, w, h, ...)]
+    Returns:
+        iou: shape is [num_boxes]
+    """
+    assert boxes.size != 0, "Cannot calculate if ground truth boxes is zero"
```

Comment:
don't use assert to test if the function is used correctly.
use `if` and raise a `ValueError`

Reply:
It is a legacy code, I changed nothing but place. Need I change this function body itself? 🤔